sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisTool.java

```diff
@@ -133,11 +146,39 @@ private static boolean isCsv(Path outputFile) {
         }
     }
 
+    private static String buildContingencyStatusPath(boolean csv, String outputFile) {
+        if (csv) {
+            return outputFile.replace(".csv", "ContingencyStatus.csv");
```

Comment:
Maybe something with a _ would be more clean like
toto.csv -> toto_contingency_status.csv ?
toto.json -> toto_contingency_status.json ?